apps/api-extractor/src/collector/VisibilityChecker.ts

```diff
@@ -0,0 +1,101 @@
+import * as path from 'path';
+import * as ts from 'typescript';
+
+import { Collector } from './Collector';
+import { AstSymbol } from '../analyzer/AstSymbol';
+import { AstDeclaration } from '../analyzer/AstDeclaration';
+// import { SymbolMetadata } from './SymbolMetadata';
```

Comment:
Remove?

Reply:
I was starting to implement consistency checking (e.g. an `@beta` function should not return an `@alpha` type), but I think I'll do that as a separate PR.